src/styles/blocks/main-block.scss

```diff
@@ -0,0 +1,28 @@
+.main {
+  &__section {
+    padding: 0;
+    margin: 0;
+  }
+}
+
+.product-cards {
```

Comment:
neither `main` nor `product-cards` are `main-block`. Filename should be exactly block name, and only this block should be inside